tests/coalaCITest.py

```diff
@@ -19,38 +19,44 @@ def tearDown(self):
         sys.argv = self.old_argv
 
     def test_log(self):
-        retval, output = execute_coala(
+        retval, stdout, stderr = execute_coala(
             coala_ci.main, 'coala-ci', '--help')
+        output = stdout + '\n' + stderr
```

Comment:
This defeats the purpose of separating stdout and stderr.
The objective is to be properly testing both streams.